js/chips.js

```diff
@@ -51,15 +52,49 @@ function isEnemyChip(selectedChip, evalChip) {
 	// return a boolean
 }
 
+function userClick(element) {
+	//takes a parameter of e.target, all of the elements on the page
+	
+	//if element is chip...
+	if (element.hasClass('chip')) {
```

Comment:
I've made some edits to this branch to cleanup some syntax breaks, but this is a good place for you to dive into some more javaScript... the `.hasClass()` method is from the jQuery API, but we're not using jQuery in this project (don't want to use it because of the eventual transition to VueJS). 

Here you can read about how this would be implemented in jQuery: https://www.w3schools.com/jquery/html_hasclass.asp

See if you can re-write the instances of `.hasClass` using javaScript's `indexOf()` method in conjunction with the `.classList` array: https://www.w3schools.com/jsref/jsref_indexof_array.asp

Reply:
also, before working on indexOf, you'll want to `git pull` my changes in this branch to your local instance

Reply:
Will do. Thanks for the fixes!

Reply:
Had to go with `if (element.classList.contains('chip'))`
because indexOf seems to work only with strings, or at least not with the array that classList returns.